go-fuzz-build/main.go

```diff
@@ -40,6 +40,7 @@ var (
 	flagLibFuzzer = flag.Bool("libfuzzer", false, "output static archive for use with libFuzzer")
 	flagBuildX    = flag.Bool("x", false, "print the commands if build fails")
 	flagPreserve  = flag.String("preserve", "", "a comma-separated list of import paths not to instrument")
+	flagGoCmd     = flag.String(*flagGoCmd, *flagGoCmd, "path to \"go\" command")
```

Comment:
Should this be = flag.String(“go-bin”, “go”, “path to Go binary”)?

Reply:
> Should this be = flag.String(“go-bin”, “go”, “path to Go binary”)?

I just follow what `compilebench` does https://pkg.go.dev/golang.org/x/tools/cmd/compilebench

Reply:
I concur with @odeke-em. And I'd rather call the flag "go" instead of "go-bin". Which appears to be what compilebench does: https://github.com/golang/tools/blob/v0.1.0/cmd/compilebench/main.go line 105.

Reply:
> I concur with @odeke-em. And I'd rather call the flag "go" instead of "go-bin". Which appears to be what compilebench does: https://github.com/golang/tools/blob/v0.1.0/cmd/compilebench/main.go line 105.

I'm somewhat confused. The change is already used "go" instead of "go-bin", and what @odeke-em suggested is "go-bin"

Reply:
What Josh means is please use “go” inlined instead of “go-bin” that I suggested, and instead of *flagGoCmd

Reply:
Yes, sorry, I meant the string literal “go” in both places.

Reply:
Ah right, done. My bad, "search and replace" problem.